event/producer_test.go

```diff
@@ -0,0 +1,125 @@
+package event_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	kafka "github.com/ONSdigital/dp-kafka/v2"
+	"github.com/ONSdigital/dp-kafka/v2/kafkatest"
+	"github.com/ONSdigital/dp-search-data-extractor/event"
+	"github.com/ONSdigital/dp-search-data-extractor/event/mock"
+	"github.com/ONSdigital/dp-search-data-extractor/models"
+	"github.com/ONSdigital/dp-search-data-extractor/schema"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+var (
+	ctx = context.Background()
+
+	expectedSearchDataImportEvent = models.SearchDataImport{
+		DataType:        "testDataType",
+		JobID:           "",
+		SearchIndex:     "ONS",
+		CDID:            "",
+		DatasetID:       "",
+		Keywords:        []string{""},
+		MetaDescription: "",
+		Summary:         "",
+		ReleaseDate:     "",
+		Title:           "",
+	}
+)
+
+func TestProducer_SearchDataImport(t *testing.T) {
+
```

Comment:
Ideally I don't think there should be a blank line here either - the Convey should come straight after the func signature if there's nothing else between the two.